src/client/Microsoft.Identity.Client/OAuth2/DeviceAuthJWTResponse.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Runtime.Serialization;
+using Microsoft.Identity.Client.Utils;
+
+namespace Microsoft.Identity.Client.OAuth2
+{
+    [DataContract]
```

Comment:
Please rebase your change and start using the newtonsoft.json attributes (DataContract -> JsonObject, DataMember -> JsonProperty). These also work with newtonsoft, but it be more consistent to use the native ones.